php/seatreg_functions.php

```diff
@@ -3602,4 +3604,18 @@ function seatreg_create_payment_log() {
 	}else {
 		wp_send_json_error();
 	}
-}
\ No newline at end of file
+}
+
+register_activation_hook(__FILE__, 'add_plugin_capabilities_on_activation');
+function add_plugin_capabilities_on_activation() {
+    $role = get_role('administrator');
+    $role->add_cap('manage_events');
+    $role->add_cap('manage_bookings');
+}
+
+register_deactivation_hook(__FILE__, 'remove_plugin_capabilities');
+function remove_plugin_capabilities() {
+	$role = get_role('administrator');
+	$role->remove_cap('manage_events');
+	$role->remove_cap('manage_bookings');
+}
```

Comment:
Can we prefix the capability names with `seatreg_`? Just to be sure we wont remove capabilities added by other plugins.